collect_app/src/main/java/org/odk/collect/android/analytics/AnalyticsEvents.java

```diff
@@ -176,12 +176,21 @@ private AnalyticsEvents() {
     public static final String TRY_DEMO = "TryDemo";
 
     /**
-     * Tracks how often projects are created using QR codes
-     */
+     * Tracks how often projects are created using QR codes. he total number of
+     * created projects can be calculated from the combined count of
+     * {@link AnalyticsEvents#MANUAL_CREATE_PROJECT} and {@link AnalyticsEvents#QR_CREATE_PROJECT}.
```

Comment:
Trying a demo and configuring with Google Drive also create projects so maybe this comment is a little misleading? Creation from qr code image also isn't being tracked so we wouldn't be able to get a complete project count. I don't feel that we need to keep tracking that path since we've been tracking it for a while and we know it's important for those who need it but for most folks it's not necessary.

Does total project count seem meaningful to you?